src/main/resources/graphql/flashcards.graphqls

```diff
@@ -0,0 +1,32 @@
+"""
+A flashcard is a set of two or more sides. Each side has a label and a text.
+The label is used to specify which side of the flashcard is being shown to the user first for learning
+and which sides he has to guess.
+"""
+type Flashcard {
+    """
+    Unique identifier of this flashcard.
+    """
+    id: UUID!
+    """
+    List of sides of this flashcard.
+    """
+    sides: [FlashcardSide!]!
+    """
+    The labels of the sides that are shown to the user first (as a question).
+    """
+    questionLabels: [String!]!
+    """
+    Last time this flashcard was learned by the user. Can be null if the user has never learned this flashcard.
+    """
+    lastLearned: DateTime
```

Comment:
This needs to be stored in some kind of user data store, not directly in the flashcard

Reply:
True

Reply:
Can stay if we store the user data in the service and if we're fine that these fields are filled automatically from the service-internal user store depending on the passed user id in the query context.

Reply:
I would at least put it in  a sub object `userData` or something similar

Reply:
yea

Reply:
I just removed it for now